Bindings/bindings.json

```diff
@@ -475,6 +475,22 @@
                     "required": true,
                     "label": "Service Bus connection",
                     "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights."
+                },
```

Comment:
Have an issue filed in portal for it. 
https://github.com/projectkudu/AzureFunctionsPortal/issues/329

resource=EventHub actually opens service bus connection blade. It needs to be renamed.